analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AbstractTypesShouldNotHaveConstructors.CSharp12.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace CSharp12
+{
+    abstract class PrimaryConstructor(int i) // FN - Primary constructor is public by default
```

Comment:
Since with primary constructors there is no way of dictating the accessibility. I would not raise here. Primary constructors if I understand correctly are used to let users write less code, so if we force a user not to use this feature one would need to write a lot more code to achieve the same.